### DISCUSSION TITLE
Rider keyboard shortcuts for Copilot not working

### DISCUSSION THREAD

**Discussion by p4nu** [community/community#7342] [Copilot Conversations] [answered]
Hey.

I have nordic layout keyboard and edited shortcuts for (alt + ä) and (alt + ö) for the next and previous suggestion, but those will not work at all. The original shortcuts are not possible for nordic keyboards. only shortcut I managed to get working is the Show completions shortcut (alt + plus)

The shortcuts work in vscode.

![image](https://user-images.githubusercontent.com/28947061/140468905-be3a0307-2a54-4854-a4de-9d696403e894.png)

**Comment by jansorg** [comment, marked as answer]
Thanks for reporting! 
Which file type were you editing? Currently, alternative completions are only available in Python and Java files. But this will be improved with one of the next updates.

**Reply by p4nu** [reply]
I'm editing C#. I can see the alternative completions when opening the copilot tab. 😕

![image](https://user-images.githubusercontent.com/28947061/140469406-955b573c-0795-4269-9c68-efb4e81304a2.png)

**Reply by jansorg** [reply]
Thanks! In this case, the update mentioned above will fix this for you. The alternative suggestions in the editor (i.e. not in "Open Copilot") are currently unavailable for C#.

**Reply by p4nu** [reply]
Sorry for the misunderstanding 😅

**Comment by broomfn** [comment]
Has this been fixed, does this work now?

**Comment by andersnicolai** [comment]
I also have a nordic keyboard and a Mac, we need a way to change keyboard shortcut for suggestions in Github copilot please
I also tried changing the keymap in settings in Rider, that did not work
